src/components/icons/ArrowRight.tsx

```diff
@@ -0,0 +1,22 @@
+import { SVGAttributes } from "react";
+
+const ArrowRight = (props: SVGAttributes<SVGElement>) => (
```

Comment:
how are we styling the icons (color, bg)?

Reply:
We don't have this option and we are using color using the icon component
We only have one color for these set of icons

Reply:
May be we can have the currentColor instead of hardcoding the colors

Reply:
just `color` defaulting to the existing one

Reply:
same thing with `background` when applicable

Reply:
@fneves We have a wrapper that does this in Icon.tsx

```

const SvgWrapper = styled.div<Partial<IconProps>>`
  display: flex;
  align-items: center;

  & path[stroke] {
    stroke: ${(props) => props.color || "currentColor"};
  }

  & path[fill] {
    fill: ${(props) => props.color || "currentColor"};
  }

  & svg {
    width: ${(props) =>
      props.width ||
      props.theme.click.image[props.size || "medium"].size.width ||
      "24px"};
    height: ${(props) =>
      props.height ||
      props.theme.click.image[props.size || "medium"].size.height ||
      "24px"};
  }

  & svg[stroke] {
    stroke: ${(props) => props.color || "currentColor"};
  }

  & svg[fill]:not([fill="none"]) {
    fill: ${(props) => props.color || "currentColor"};
  }
`;

```

Should we change the behaviour?

Reply:
I see. This works. color is the prop. `currentColor` is the css property to inherit from wrapping element. This should work if all icons are wrapped in this.